languages/ruby/exercises/concept/exceptions/.docs/introduction.md

```diff
@@ -0,0 +1,73 @@
+Exceptions in Ruby as in most languages provide a way of dealing with unexpected events. Proper handling of exceptions is important when trying to prevent your program from crashing.
+
+When an exception is raised in your code, either by raising it explicitly or by Ruby interpreter raising it itself. The program stops and eventually exit with an error message:
+
+```ruby
+raise ArgumentError.new("Something went wrong!")
+=> Traceback (most recent call last):
+.
+.
+ArgumentError (Something went wrong!)
+```
+
+```ruby
+1/0
+=> Traceback (most recent call last):
+.
+.
+ZeroDivisionError (divided by 0)
+```
+
+In case we want to stop this shut down process we need to react to the error. This is called "rescuing" and exception:
+
+```ruby
+begin
+  # ...any code that raises an exception
+rescue
+  puts 'Got an exception'
```

Comment:
We might mention here that we can output to standard error rather than standard out. (Not necessarily this line, but this line specifically reminds me about it, since it is going to standard out, which may not be desirable.

Reply:
I don't think this information is something that necessarily belongs to the introduction, as it is just used to demonstrate the syntax. So my suggestion would be to only mention the standard error in the concept document (see https://github.com/exercism/v3/issues/2293).